src/sentry/tasks/beacon.py

```diff
@@ -14,33 +14,49 @@
 from sentry.tasks.base import instrumented_task
 from sentry.utils import json
 
-BEACON_URL = "https://sentry.io/remote/beacon/"
+#  BEACON_URL = "https://sentry.io/remote/beacon/"
+BEACON_URL = "https://dev.getsentry.net:8000/remote/beacon/"
```

Comment:
Was this intentional? Looks like a mistake to me.

Reply:
Ah shit, definitely not intentional, was using this to test.

Reply:
Fixed in #26072 ftr